mesonbuild/minit.py

```diff
@@ -176,7 +176,7 @@ def run(options: 'argparse.Namespace') -> int:
             print('Build directory already exists, deleting it.')
             shutil.rmtree(options.builddir)
         print('Building...')
-        cmd = mesonlib.get_meson_command() + [options.builddir]
+        cmd = T.cast(T.List[str], mesonlib.get_meson_command()) + [options.builddir]
```

Comment:
I wouldn't cast here, it's unlikely this is null, but it could be. I'd probably do
```python
cmd = mesonlib.get_meson_command()
assert cmd is not None, 'for mypy'
cmd = cmd + [options.builddir]  # to avoid mutating cmd
```